File: pset1/Mario-less comfortable/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    do
    {
        size=get_int("Height: ");
    }
    while (size>8 || size<1);

    for (int counter_one=0; counter_one<size; counter_one++)
    {
        for (int counter_two=0; counter_two<size ;counter_two++)
        {
            if(counter_one+counter_two<size-1)
                printf(" ");
            else
                printf("#");
        }
        printf("\n");
    }
}

File: pset6/Mario/less/mario.py
from cs50 import get_int


def main():
    size = get_size()
    for i in range(size):
        print(" " * (size - 1 - i), end = '')
        print("#" * (i + 1))


def get_size():
    while True:
        size = get_int("Height: ")
        if size > 0 and size < 9:
            break
    return size

main()

